ui/tests/acceptance/allocation-detail-test.js

```diff
@@ -219,7 +219,9 @@ module('Acceptance | allocation detail', function(hooks) {
     await Allocation.visit({ id: 'not-a-real-allocation' });
 
     assert.equal(
-      server.pretender.handledRequests.findBy('status', 404).url,
+      server.pretender.handledRequests
+        .reject(request => request.url.includes('policy'))
```

Comment:
What are your thoughts on phasing out Ember's array extension methods? In this case, it seems simple enough to stick with `filter`.

Reply:
No problem, I sometimes forget what’s an Ember thing and what isn’t, but it was indeed trivial to convert.